### ISSUE TITLE
change greetd package to UsrEtc config

### ISSUE THREAD

**Issue by denisok** [openSUSE/openSUSEway#58]
Change config location to `/usr/etc` for `greetd` in openSUSE.

https://todo.sr.ht/~kennylevinsen/greetd/20

actually we can pass `--config` as the parameter in service file:
https://git.sr.ht/~kennylevinsen/greetd/tree/master/item/greetd.service#L9

to support `/usr/etc` config instead of `/etc/`: https://build.opensuse.org/package/show/X11:Wayland/greetd

**Comment by denisok** [comment]
@danyspin97 maybe you could help with this one ?

**Comment by denisok** [comment]
It is nice to have for the MicroOS and similar OSes where `/etc` is reserved to the base OS. It would be cool to have all openSUSEway config files under `/usr/etc` instead.

Just goo improvement, but not a critical one.

**Comment by LakshayPuhal** [comment]
I would like to work on this issue.

**Comment by FilippoBonazziSUSE** [comment]
I think greetd and associated greeters (gtkgreet, ...) would need to be patched to look for files under /etc/greetd first (current behaviour), then under /usr/etc/greetd (see https://en.opensuse.org/openSUSE:Packaging_UsrEtc).

greetd would need to be patched when looking for `/etc/greetd/config.toml`, gtkgreet would need to be patched when looking for `/etc/greetd/environments`. Perhaps there are others.

This could be done in OBS right away, and proposed to upstream in parallel.

Then we could provide our files under `/usr/etc` as `%config` (without `noreplace`) without fear of messing with the user-managed files under `/etc`.
